modules/axios/plugin.js

```diff
@@ -135,6 +135,15 @@ export default (ctx) => {
   });
   <% } %>
 
+  <% if (options.requestInterceptor) { %>
+  // Custom request interceptor
+  const reqInter = <%= options.requestInterceptor %>
+  axios.interceptors.request.use(
+    (config) => reqInter(config, store),
```

Comment:
What's your idea passing full `ctx` instead of just store? (And also maybe move config to the second param to keep consistent with other nuxt functions which ctx is the first argument)